variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "gce-region" {
+  default = "insert region"
+}
+
+variable "gce-project" {
+  default = "insert project"
+}
+
+
+variable "k8s-context" {
+}
+
+variable "helm_cluster_admin_tls_client_key" {
+}
+
+variable "helm_cluster_admin_tls_client_cert" {
+}
+
+variable "helm_cluster_admin_tls_ca_path" {
+}
+
+variable "namespace" {
+  default = "kube-system"
+}
+
+variable "spotinst_account" {
+    default = ""
+}
+
+variable "spotinst_token" {
+    default = ""
+}
```

Comment:
Terraform repo have to have pre-commit hooks for terraform. 
https://github.com/antonbabenko/pre-commit-terraform

Module have to have proper versioning with sem ver: 1.0.0 like etc